src/screens/Products/AddProduct.js

```diff
@@ -0,0 +1,310 @@
+import { AddImage, AsyncButton, UserInput, Wallpaper } from '@components';
+import React, { Component } from 'react';
+import { StyleSheet, ScrollView, View, Text, TextInput, Picker } from 'react-native';
+import ImagePicker from 'react-native-image-crop-picker';
+
+export default class AddProduct extends Component {
+  static navigationOptions = {
+    title: 'Add a Handmade Product'
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      productMainCategory: 'Handmade Apparel',
+      productSubCategory: 'Activewear',
+
+      productMainPicture: "",
+      productTitle: "",
+      productDescription: "",
+      productGender: 'Women',
+      productStandardPrice: "",
+      productSellerSKU: "",
+      productQuantity: "",
+      productTime: "",
+
+      //If you adding product, assumes never sold before
+      productTimesSold: 0,
+      focusedInputs: {title: false, desc: false, price: false, sku: false, quantity: false, time: false},
```

Comment:
Orange highlighting around the box, for when a text input field is selected (focused = true), this is an interesting feature.